Sources/OpenCombineFoundation/OperationQueue+Scheduler.swift

```diff
@@ -187,7 +187,8 @@ extension OperationQueue {
             }
 
             private func becomeReady() {
-#if canImport(Darwin)
+// Smart key paths don't work with NSOperation in swift-corelibs-foundation prior to Swift 5.1.
+#if canImport(Darwin) || swift(>=5.1)
```

Comment:
I think you've meant this:

```suggestion
#if canImport(Darwin) || swift(<5.1)
```